Source/swiftlint/StructuredText.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  StructuredText.swift
+//  SwiftLint
+//
+//  Created by Chris Eidhof on 20/05/15.
+//  Copyright (c) 2015 Realm. All rights reserved.
+//
+
+import Foundation
+
+typealias StructuredInlineText = String
+
+private extension Array {
+    func dropWhile(x: Element -> Bool) -> [Element] {
+        var count = 0
+        while x(self[count]) { count++ }
+        var copy = self
+        copy.removeRange(0..<count)
+        return copy
+    }
+}
+
+
+extension String {
+    var chomped: String {
+        return String(reverse(self).dropWhile { $0 == "\n" }.reverse())
```

Comment:
Could you use `String.lines()` for this instead? https://github.com/realm/SwiftLint/blob/0.1.0/Source/SwiftLintFramework/String+SwiftLint.swift#L13-L20